bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/QuantityType.java

```diff
@@ -74,7 +74,7 @@ public class QuantityType<T extends Quantity<T>> extends Number
     // split on any blank character, even none (\\s*) which occurs after a digit (?<=\\d) and before
     // a "unit" character ?=[a-zA-Z°µ%'] which itself must not be preceded by plus/minus digit (?![\\+\\-]?\\d).
     // The later would be an exponent from the scalar value.
-    private static final String UNIT_PATTERN = "(?<=\\d)\\s*(?=[a-zA-Z°µ%'](?![\\+\\-]?\\d))";
+    private static final String UNIT_PATTERN = "(?<=\\d)\\s*(?=[a-zA-Z°µ\u03BC\u00B5%'](?![\\+\\-]?\\d))";
```

Comment:
Can you also update the comment and add tests for this change?

Reply:
Updated. There is a test in UnitTests.java that also check for `\u03BC` but not for `\u00B5`. It caught the fact that the regex failed to split up the unit. 

However, I feel that this regex pattern is too restrictive. Although I don't have a specific case where it would fail, the fact that it failed because it didn't account for the special unicode character for "micro" here makes me wonder if there would be some other esoteric but valid unicode characters it would also fail on.

I tried this other pattern that's more broad, and it seems to pass the current tests too: 
`"(?<=\\d)\\s*(?=\\D(?![\\+\\-]?\\d))"`
However, I hesitate to use it here pending further testing. Is it worth pursuing, perhaps in a separate PR? Nobody has ever complained about it before, so I'd probably rather just let it be.

For the purpose of this PR, the current pattern in the PR should suffice and it at least sticks to the way it had been working before.

Reply:
> Is it worth pursuing, perhaps in a separate PR? Nobody has ever complained about it before, so I'd probably rather just let it be.

I would also only do that when neccessary. :wink: 

But isn't there duplication in the pattern now? The µ seems to be the same character as `\u00B5`:

I see `true` when I run:

```java
System.out.println("µ".equals("\u00B5")); 
```